components/centraldashboard/package.json

```diff
@@ -61,6 +61,7 @@
     "clean-webpack-plugin": "^1.0.1",
     "concurrently": "^4.1.0",
     "copy-webpack-plugin": "^5.0.0",
+    "cross-env": "^5.2.0",
```

Comment:
I also noticed that the terser-webpack-plugin got added to deps instead of devDeps. Can you move that in this commit?

Reply:
Sure